docs/sources/plugins/developing/development.md

```diff
@@ -28,21 +28,18 @@ Example plugins
 - [Clock panel](https://github.com/grafana/clock-panel)
 - [Pie chart panel](https://github.com/grafana/piechart-panel)
 
-There are two blog posts about authoring a plugin that might also be of interest to any plugin authors.
+There are two blog posts about authoring a plugin that might also be of interest to other users.
 
 - [Timing is Everything. Writing the Clock Panel Plugin for Grafana](https://grafana.com/blog/2016/04/08/timing-is-everything.-writing-the-clock-panel-plugin-for-grafana-3.0/)
 - [Timing is Everything. Editor Mode in Grafana for the Clock Panel Plugin](https://grafana.com/blog/2016/04/15/timing-is-everything.-editor-mode-in-grafana-3.0-for-the-clock-panel-plugin/).
 
 ## What languages?
 
-Since everything turns into javascript it's up to you to choose which language you want. That said it's probably a good idea to choose es6 or typescript since
-we use es6 classes in Grafana. So it's easier to get inspiration from the Grafana repo if you choose one of those languages.
+Since everything turns into javascript it's up to you to choose which language you want. That said, it's probably a good idea to choose es6 or typescript since we use es6 classes in Grafana. So it's easier to get inspiration from the Grafana repo if you choose one of those languages.
 
 ## Buildscript
 
-You can use any build system you like that support systemjs. All the built content should end up in a folder named ```dist``` and committed to the repository.
-By committing the dist folder the person who installs your plugin does not have to run any buildscript.
-All our example plugins have build scripted configured.
+You can use any build system you like that supports systemjs. All the built content should end up in a folder named ```dist``` and committed to the repository. By committing the dist folder the person who installs your plugin does not have to run any build script. All of our example plugins have build script configured.
```

Comment:
```suggestion
You can use any build system that supports systemjs. All the built content should end up in a folder named ```dist``` and be committed to the repository. By committing the dist folder, the person who installs your plugin does not have to run any build script. All of our example plugins have a build script configured.
```

Reply:
Do we want to mention that a README in the dist folder overrides the README on the Plugins website? @DanCech ?